site/installing.md

```diff
@@ -37,9 +37,13 @@ This script downloads and runs a recent Scope image from the Docker Hub. Scope n
 
 After it’s been launched, open your browser to `http://localhost:4040`.
 
-> **Note:** If you're using boot2docker, replace localhost with the output of boot2docker IP.
+If you are using docker-machine, replace localhost with the IP after you've run, `docker-machine ip <VM name>`.
```

Comment:
You could also mention that the scope script prints the URLs scope is available at, so they could try one of those.